inception-pdf-editor/src/main/js/pdfanno/src/page/pdf/PDFAnnoPage.js

```diff
@@ -613,37 +613,42 @@ export default class PDFAnnoPage {
       method : 'GET',
       mode   : 'cors'
     }).then(response => {
+// BEGIN PDFANNO EXTENSION - #624 - Integration of PDFExtractor
```

Comment:
Within such a change-marking block, I'd recommend to keep the entire original code in a `/* */` block - that makes it easier to see what changed when merging in upstream changes. Cf. visualizer.js in WebAnno.

Reply:
I changed the extensions.
Is it fine like this?